kratos/includes/model_part_io.h

```diff
@@ -713,11 +713,9 @@ class KRATOS_API(KRATOS_CORE) ModelPartIO : public IO
 
     char SkipWhiteSpaces();
 
-    bool IsWhiteSpace(char C);
-
     char GetCharacter();
 
-    bool CheckStatement(std::string const& rStatement, std::string const& rGivenWord);
```

Comment:
this function declaration, along with the rest of the `private` crap only used inside this class, could be just moved to source as well.

Reply:
fully agree

You ok if I do it in a separate PR?

Reply:
of course